website/gatsby-config.js

```diff
@@ -9,8 +9,12 @@ module.exports = {
       {
         label: 'Getting started',
         subgroup: [
-          { label: 'Installation', linkTo: '/getting-started/installation' },
+          { label: 'Introduction', linkTo: '/getting-started/introduction' },
```

Comment:
I'm still not 100% sure about this structure/grouping, will just keep it like this for now though